backend/program/debrid/realdebrid.py

```diff
@@ -15,9 +17,25 @@ class Debrid:  # TODO CHECK TORRENTS LIST BEFORE DOWNLOAD, IF DOWNLOADED AND NOT
     """Real-Debrid API Wrapper"""
 
     def __init__(self):
-        self.settings = settings_manager.get("debrid_realdebrid")
-        self.auth_headers = {"Authorization": f'Bearer {self.settings["api_key"]}'}
-        self._torrents = {}
+        # Realdebrid class library is a necessity
+        while True:
```

Comment:
Don't agree with the continuous validation of RD. Since the app is meant for RD, would it be better to just exit the app if the first validation fails for this?

Reply:
@Gaisberg